Karma-Khameleons/bangazon-website
# Sam payment type creation

## Description
Includes an initialization of the PaymentType model, along with a set of tests to confirm that module functions correctly.
Also reconfigures admin.py a little bit because the other method was giving me problems.

## Related Ticket(s)
#4 

## Problem to Solve
A PaymentType model needs to be in place for Customers to create new payment types

## Changes
PaymentType model created
admin.py file reconfigured using a different register method


## Expected Behavior
PaymentType instances should be able to be created from the Django admin page.


## Steps to Test Solution
1. Go to the Django admin page
1. Create a new PaymentType
1. Open up the database in DB browser
1. Ensure that new PaymentType is saved to the database with all correct values


##Unit Testing
[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

##Documentation

[x] There is new documentation in this pull request that must be reviewed..
[x] I added documentation for any new classes/methods

##Deploy Notes
Run the following commands from the Django project root
```bash
git fetch --all
git checkout sam-paymentTypeCreation
python manage.py migrate
python manage.py test bang_app
```
Both 
```test_new_payment_is_of_class_PaymentType ```
and 
```test_payment_fields_contain_correct_values```
should pass
